mac/hid.c

```diff
@@ -1203,6 +1203,18 @@ HID_API_EXPORT const wchar_t * HID_API_CALL  hid_error(hid_device *dev)
 	return L"hid_error is not implemented yet";
 }
 
+int HID_API_EXPORT hid_init_sixaxis_usb(hid_device *dev)
```

Comment:
Missing ifdef

Reply:
Is it necessary? The entire file is Mac-specific.

Reply:
Oh. Didn't see that